package.json

```diff
@@ -40,22 +40,25 @@
     "build-electron-all": "npm run clean && npm run build-electron -- --platform=win32 && npm run build-electron -- --platform=linux && npm run build-electron -- --platform=darwin",
     "clean": "rm -rf builds config.json tasks.json coverage dist",
     "lint": "eslint *.js app test",
-    "dist": "npm run clean && electron-builder -wlm"
+    "dist": "npm run clean && electron-builder -wlm --x64 --ia32",
+    "dist-linux": "npm run clean && electron-builder -l --x64 --ia32",
+    "dist-win": "npm run clean && electron-builder -w --x64 --ia32",
+    "dist-mac": "npm run clean && electron-builder -m"
```

Comment:
--x64 and --ia32 are not needed here?

Reply:
It's just to build for both x64 and x86 architectures. By default it will only build for the architecture you are running at compile-time (usually x64).

Reply:
Yeah, I know, my question is why not using it in `dist-mac`?

Reply:
macOS just works on 64 bits.